docker/README.md

```diff
@@ -0,0 +1,31 @@
+# Iroh Docker Images
+
+## Intro
+
+A set of docker images provided to easily run iroh in a containerized environment.
+Features `iroh`, `iroh-relay` and `iroh-dns-server`.
+
+The provided `Docker` files are intended for CI use but can be also manually built.
+
+## Building
+
+- All commands are run from the root folder
+- If you're on macOS run `docker buildx build -f docker/Dockerfile --target iroh --platform linux/arm64/v8 --tag n0computer/iroh:latest .`
```

Comment:
what does `buildx` mean?

Reply:
Buildx is an official extension by docker for the build system for images. Gives you the ability to specify platforms and some other nifty stuff.